feat(module:splitter): add splitter component (#8987)
NG-ZORRO/ng-zorro-antd: components/splitter/doc/index.en-US.md

---
category: Components
type: Layout
title: Splitter
tag: 19.2.0
cover: https://mdn.alipayobjects.com/huamei_7uahnr/afts/img/A*f0SISaETY0wAAAAAAAAAAAAADrJ8AQ/original
---

> Split panels to isolate.

Reviewer: ```suggestion
Splits the area into multiple horizontal or vertical sides.
```

## When To Use

> Can be used to separate areas horizontally or vertically. When you need to freely drag and adjust the size of each area.
> When you need to specify the maximum and minimum width and height of an area.

Reviewer: ```suggestion
The Splitter component can be used to separate areas horizontally or vertically.  
You can freely drag the splitter to adjust the size of each area, while also being able to specify its maximum and minimum width and height.
```

```ts
import { NzSplitterModule } from 'ng-zorro-antd/splitter';
```

## API

### nz-splitter

> | 参数                | 说明                              | 类型                           | 默认值            | 全局配置 |

Reviewer: Column titles should be in English.
